minibroker/minibroker-master.yaml

```diff
@@ -0,0 +1,293 @@
+---
+shared:
+  helm-charts-s3-configs: &helm-charts-s3-configs
+    access_key_id: ((s3-access-key))
+    bucket: ((s3-bucket))
+    disable_ssl: ((s3-disable-ssl))
+    endpoint: ((s3-endpoint))
+    private: false
+    secret_access_key: ((s3-secret-key))
+  helm-charts-src-configs: &helm-charts-src-configs
+    branch: ((helm-charts-src-branch))
+    private_key: ((github-private-key))
+    uri: ((helm-charts-src-repo-ref))
+  s3-configs: &s3-configs
+    access_key_id: ((s3-access-key))
+    bucket: ((s3-bucket))
+    disable_ssl: ((s3-disable-ssl))
+    endpoint: ((s3-endpoint))
+    private: false
+    secret_access_key: ((s3-secret-key))
+  src-ci-configs: &src-ci-configs
+    uri: ((ci-repo))
+    branch: ((ci-branch))
+    private_key: ((github-private-key))
+
+
+resources:
+- name: src
+  type: git
+  source:
+    branch: ((src-branch))
+    private_key: ((github-private-key))
+    uri: ((src-repo-ref))
+
+- name: src-redis
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/redis
+
+- name: src-postgres
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/postgresql
+
+- name: src-mariadb
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/mariadb
+
+- name: src-mongodb
+  type: git
+  source:
+    <<: *helm-charts-src-configs
+    paths:
+    - stable/mongodb
+
+- name: src-ci
+  type: git
+  source:
+    <<: *src-ci-configs
+    paths:
+    - minibroker
+
+- name: src-ci-docker
+  type: git
+  source:
+    <<: *src-ci-configs
+    paths:
+    - minibroker/Dockerfile
```

Comment:
Why don't we need this in the -check pipeline?

Reply:
The check pipeline is using the docker image generated in the master pipeline. It doesn't change very often, so I didn't think it needed a separate Docker image - otherwise we'd have to keep track of the two Docker images (or different tags?) and pull from a different one depending on the pipeline.

Maybe the Docker image generation should be a separate pipeline, and the two pipelines just refer to the Docker image?